core/kazoo_apps/priv/couchdb/account/users.json

```diff
@@ -35,7 +35,7 @@
             "map": [
                 "function(doc) {",
                 "  if (doc.pvt_type != 'user' || doc.pvt_deleted) return;",
-                "  var features = [];",
+                "  var features = doc.features;",
```

Comment:
maybe `var features = doc.features || [];`

Reply:
just thought of that myself :)